### PR TITLE
[EDA] Metadata API Refresh - Metecho Readiness

### PR DESCRIPTION
**Pull request by skosgi1**
# Critical Changes

# Changes

# Issues Closed

# New Metadata

## Unpackaged Metadata

# Deleted Metadata

# Testing Notes

### CODE REVIEW

File: src/objectTranslations/Address__c-ca.objectTranslation [hunk 1]
```diff
@@ -1,481 +1,487 @@
...
+        <plural>false</plural>
+        <value><!-- Address --></value>
+    </caseValues>
+    <caseValues>
+        <plural>true</plural>
+        <value><!-- Address --></value>
+    </caseValues>
+    <fields>
+        <help>Seleccioneu quin tipus d’adreça es tracta, per exemple, de casa, de la feina, de la residència de vacances o d’una altra.</help>
+        <label>Tipus d&apos;adreça</label>
```
Comment (skosgi1) [inline src/objectTranslations/Address__c-ca.objectTranslation:13]: @MatthewBlanski is this change ok in the label?

Comment (MatthewBlanski) [reply src/objectTranslations/Address__c-ca.objectTranslation:13]: Replacement of apostrophes with apos are acceptable. This is a platform function.

File: src/objectTranslations/Address__c-de.objectTranslation [hunk 2]
```diff
@@ -1,522 +1,518 @@
...
-  <layouts>
-    <layout>HEDA Address Layout</layout>
-    <sections>
-      <label>Benutzerdefinierte Links</label>
-      <section>Custom Links</section>
-    </sections>
-  </layouts>
-  <nameFieldLabel>Adressen-ID</nameFieldLabel>
-  <standardFields>
-    <name>Name</name>
```
Comment (skosgi1) [inline src/objectTranslations/Address__c-de.objectTranslation:478]: @MatthewBlanski looks like this one is missing here, I will check all the translations files again and push the changes in a single commit.

Comment (skosgi1) [reply src/objectTranslations/Address__c-de.objectTranslation:478]: @MatthewBlanski Only after doing a few I realized, looks like <standardFields> <name>...</name></standardField> is replaced with <nameFieldLabel>....</nameFieldLabel>, that is causing the difference in line count.

**Review by MatthewBlanski — APPROVED** [review]
Builds and compiles! Woot!

**Pull request merged**